languages/javalang/builder-renderer/java-builder-client/src/test/kotlin/io/vrap/codegen/languages/javalang/client/builder/BuilderTestCodeGenerator.kt

```diff
@@ -26,7 +26,7 @@ class BuilderTestCodeGenerator {
          * Specifies a path where the code generator should place generated code
          * */
         private val generatedCodePath = System.getenv("GENERATED_CODE_PATH")
-        private val apiPath : Path = Paths.get(if (userProvidedPath == null) "../api-spec/api.raml" else userProvidedPath)
+        private val apiPath : Path = Paths.get("/Users/abeniasaad/IdeaProjects/commercetools-api-reference/update-actions.raml")
```

Comment:
Please revert this change and use our `GENERATED_CODE_PATH` env var instead.

Reply:
fixed in 25da514